ocean_grid_generator.py

```diff
@@ -1539,5 +1605,8 @@ def main(
     parser.add_argument("--grids",type=str,nargs="+",required=False,default="all",
                         help="specify the subgrids to generate, choices are bipolar, mercator, so, sc, all. Default is all")
 
+    parser.add_argument("--target_ny",type=int,required=False,default="0",
```

Comment:
It would be good to add a little more info about how this is done - i.e. copying the grid metrics of the southernmost row.